dbfit-java/build.gradle

```diff
@@ -134,16 +204,39 @@ subprojects {
             }
     }
 
-    // to be overriden when needed
-    ext.connectionPropertiesFiles = []
-
     ext.copyConnectionProperties = {
         destination ->
             project.connectionPropertiesFiles.each { file ->
                 copyProperties new File(project.projectDir, file), destination
             }
     }
 
+    ext.execSqlScriptTestDatabase = {
+        scriptFile ->
```

Comment:
I vote for the one with the indentation. And also for moving the parameters on top next to `{` unless they're too many.